README.md

```diff
@@ -37,6 +37,13 @@ git clone git@github.com:SparkPost/wordpress-sparkpost.git ~/src/wordpress/wp-co
 
 * Activate the plugin from admin panel
 
+## Running Tests
+* Follow the steps in [wp-cli](http://wp-cli.org/docs/plugin-unit-tests/)
```

Comment:
It would also be nice to include phpunit via composer instead of asking people to install it separately since we're already using composer here.